### DISCUSSION TITLE
How to add support of Cutlass Simt opcode in TVM

### DISCUSSION THREAD

**Discussion by TigerYang414** [NVIDIA/cutlass#557] [Q&A] [answered]
In TVM master code, only TensorOp related opcode is integrated. How can we integrate simt to TVM?
here is TVM code snippet:

GENERATOR_FUNC_TABLE = {
    75: generate_sm75_tensor_op_1688,
    80: generate_sm80_tensor_op_16816,
}

**Comment by hwu36** [comment, marked as answer]
@masahi from tvm comunity can answer this question.  He added cutlass kernels to tvm.

**Reply by TigerYang414** [reply]
got it. I'll try in tvm community.
